airbyte-integrations/connectors/source-unleash/source_unleash/spec.yaml

```diff
@@ -0,0 +1,33 @@
+documentationUrl: https://docs.airbyte.io/integrations/sources/unleash
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Unleash Source Spec
+  type: object
+  required:
+    - api_token
+    - api_url
+  additionalProperties: true
+  properties:
+    api_token:
+      title: API Token
+      type: string
+      description: >-
+        Your API Token. See <a
+        href="https://docs.getunleash.io/how-to/how-to-create-api-tokens/">here</a>. The token is
+        case sensitive.
+      airbyte_secret: true
+    api_url:
+      title: API URL
+      type: string
+      description: >-
+        Your API URL. No trailing slash. ex: http://unleash.host.com
+    project_name:
+      title: Project Name
+      type: string
+      description: >-
+         (optional) Use this if you want to filter the API call for only one given project (can be used in addition to the "Experiment Name Prefix" field)
```

Comment:
You don't need to add (optional) the UI will display automatically.

Reply:
ah perfect, thanks, I'll remove it